samples/msal-node-samples/ElectronTestApp/src/AuthProvider.ts

```diff
@@ -118,42 +116,60 @@ export default class AuthProvider {
         return authResponse.accessToken || null;
     }
 
-    async getTokenSilent(tokenRequest: SilentFlowRequest): Promise<AuthenticationResult> {
+    async getTokenSilent(
+        tokenRequest: SilentFlowRequest
+    ): Promise<AuthenticationResult> {
         try {
-            return await this.clientApplication.acquireTokenSilent(tokenRequest);
+            return await this.clientApplication.acquireTokenSilent(
+                tokenRequest
+            );
         } catch (error) {
-            console.log("Silent token acquisition failed, acquiring token using pop up");
-            const authCodeRequest = { ...this.authCodeUrlParams, ...tokenRequest };
+            console.error(
```

Comment:
This is not an error. refresh_token grant can fail for a number of legitimate reasons like RT expiration, user changes passwrd, tenant admin adds MFA etc.

Reply:
This is out of scope for this PR. We can address this in a follow-up PR.